.vale/styles/openly/Hedging.yml

```diff
@@ -0,0 +1,29 @@
+---
+extends: existence
+message: "Consider removing '%s'"
+ignorecase: true
+level: warning
+tokens:
+  - appear to be
+  - arguably
+  - as far as I know
+  - could
```

Comment:
I can see rules like this just being annoying

There are 154 matches on the word 'could'

Reply:
Marking resolved since this is now out of scope.